### PR TITLE
Fix token weaver not reading tokens from single token studio JSON file

### LINKED ISSUE #9: Token weaver not reading tokens from single token studio JSON file
Token studio can export tokens either as single or multiple JSON files. At the moment ability to read tokens from single JSON file is broken and throws error when running the tool

### PR DESCRIPTION
**Pull request by msasikanth**
fixes #9

### CODE REVIEW

**Comment by msasikanth** [comment]
You can ignore files under `dist/` when reviewing. They are generated distribution files and we have CI check to verify that we have up-to-date dist files

File: .fleet/run.json [hunk 1]
```diff
@@ -20,5 +20,12 @@
       "program": "yarn",
       "workingDir": "$PROJECT_DIR$/"
     },
+    {
+      "type": "command",
+      "name": "Test",
+      "program": "yarn",
+      "workingDir": "$PROJECT_DIR$/",
+      "args": ["jest"],
+    },
```
Comment (sanchita) [inline .fleet/run.json]: Can we review and remove Fleet related code as its specific to a developer's config?

File: src/__test__/__snapshots__/tokens_reader.test.ts.snap [hunk 2]
```diff
@@ -5,17 +5,7 @@ exports[`read tokens should read tokens from a single file 1`] = `
...
-    {
-      "$figmaStyleReferences": {},
-      "id": "282adb2c8d3506b187c5a756a06cbebb101a31b8",
-      "name": "default",
-      "selectedTokenSets": {
-        "core": "enabled",
-        "theme/default": "enabled",
-      },
-    },
+    "/Users/sasikanth/Projects/weaver/src/temp_tokens/theme/dark.json",
```
Comment (sanchita) [inline src/__test__/__snapshots__/tokens_reader.test.ts.snap]: We seem to be using a local path somewhere. Can we verify and add a relative path?

**Review by sanchita — CHANGES_REQUESTED** [review]
Left a few comments

**Pull request merged**